### PR TITLE
fix responsiveness on mobile view

### PR DESCRIPTION
**Pull request by atulyamiglani**
1. aligned navbar items for mobile view. 
2. fixed the text alignment issue for the landing page 
3. fixed the cards for mobile view on lessonplan_find_share.html
4. fixed other issues for mobile view( email and subscribe button etc.)

### CODE REVIEW

File: curriculum/templates/curriculum/lessonplan_find_share.html [hunk 1]
```diff
@@ -48,7 +48,7 @@ <h1 class="font-weight-bold find-share-curricula">Find & Share Curricula</h1>
                             Difficulty Level
                         </a>
                     </div>
-                    <div class="pr-4">
+                    <div class="pr-4 ">
```
Comment (lukesanantonio) [inline curriculum/templates/curriculum/lessonplan_find_share.html]: remove space

File: curriculum/templates/curriculum/lessonplan_find_share.html [hunk 2]
```diff
@@ -96,11 +96,11 @@ <h1 class="font-weight-bold find-share-curricula">Find & Share Curricula</h1>
         </form>
         <div class="row my-5">
             {% for lessonplan in object_list %}
-                <div class="col-4">
+                <div class="col-md-4">
                     <div class="d-flex flex-column curriculum mb-3 card-hover-shadow border rounded border-primary p-3">
                         <a href="{{ lessonplan.get_absolute_url }}">
                             <h6 class="curriculum-title text-primary">{{ lessonplan.title }}</h6>
-                            <div class="d-flex flex-row justify-content-between">
+                            <div class="d-flex flex-row  justify-content-between">
```
Comment (lukesanantonio) [inline curriculum/templates/curriculum/lessonplan_find_share.html]: remove space

File: curriculum/templates/curriculum/lessonplan_find_share.html [hunk 3]
```diff
@@ -133,25 +133,25 @@ <h6 class="curriculum-title text-primary">{{ lessonplan.title }}</h6>
         <hr>
     </div>
     <div class="container mb-5">
-        <div class="row">
-            <div class="col-6">
-                <h6 class="font-weight-bold">Interested in sharing your curricula?</h6>
+        <div class="row justify-content-center justify-content-md-start ">
+            <div class="col-8">
+                <h6 class="font-weight-bold ">Interested in sharing your curricula?</h6>
```
Comment (lukesanantonio) [inline curriculum/templates/curriculum/lessonplan_find_share.html]: remove space

File: curriculum/templates/curriculum/lessonplan_find_share.html [hunk 4]
```diff
@@ -133,25 +133,25 @@ <h6 class="curriculum-title text-primary">{{ lessonplan.title }}</h6>
...
                 <p class="descriptions mb-5">
                     Upload your curriculum to share it with a community of
                     educators who may utilize it in their own classroom, make
                     changes for their own use, and provide feedback directly.
                 </p>
-                <a class="btn btn-primary" href="{% url 'create-lesson-plan' %}">Upload my curriculum</a>
+                <a class="btn btn-primary " href="{% url 'create-lesson-plan' %}">Upload my curriculum</a>
```
Comment (lukesanantonio) [inline curriculum/templates/curriculum/lessonplan_find_share.html]: remove space

File: curriculum/templates/curriculum/lessonplan_find_share.html [hunk 5]
```diff
@@ -133,25 +133,25 @@ <h6 class="curriculum-title text-primary">{{ lessonplan.title }}</h6>
...
-            <div class="flex-column col-4">
-                <h6 class="font-weight-bold">Subscribe to our newsletter</h6>
+        <div class="d-flex flex-row ustify-content-between align-items-center mt-5 flex-column flex-sm-row justify-content-between mb-3">
+            <div class="flex-column col-sm-4 ">
+                <h6 class="font-weight-bold ml-autos">Subscribe to our newsletter</h6>
                 <p class="description">Don't miss information about new events and volunteer opportunities! Opt out
                     anytime.</p>
             </div>
-            <div class="input-group col-5 align-self-start">
+            <div class="input-group  col-sm-5 align-self-center">
```
Comment (lukesanantonio) [inline curriculum/templates/curriculum/lessonplan_find_share.html]: remove extra space

File: landing/templates/landing/landing.html [hunk 6]
```diff
@@ -134,12 +135,12 @@ <h5 class="text-primary">micro:bit & bit:booster</h6>
...
-            <div class="d-flex flex-row justify-content-between mt-5">
-                <div class="flex-column col-4">
-                    <h6 class="font-weight-bold">Subscribe to our newsletter</h6>
+            <div class="d-flex flex-row justify-content-between align-items-center mt-5 flex-column flex-sm-row ">
+                <div class="flex-column col-sm-4 ">
+                    <h6 class="font-weight-bold ">Subscribe to our newsletter</h6>
                     <p class="description">Don't miss information about new events and volunteer opportunities! Opt out anytime.</p>
                 </div>
-                <div class="input-group col-5 align-self-start">
+                <div class="input-group  col-sm-5 align-self-center">
```
Comment (lukesanantonio) [inline landing/templates/landing/landing.html]: remove extra space

**Review by lukesanantonio — CHANGES_REQUESTED** [review]

**Comment by lukesanantonio** [comment]
Fixes #99

**Comment by lukesanantonio** [comment]
Also change breakpoints of newsletter signup to stack a little sooner. Change small to medium or medium to large or what have you.

**Comment by atulyamiglani** [comment]
Fixes #100

**Review by lukesanantonio — APPROVED** [review]
Looks great thank you so much @atulyamiglani. Merge when ready :)

**Pull request merged**